octobot_backtesting/independent_backtesting.py

```diff
@@ -120,6 +119,7 @@ def _get_market_delta(symbol, exchange_manager, min_timeframe):
     def _init_default_config_values(self):
         from octobot_trading.constants import CONFIG_TRADER_RISK, CONFIG_TRADING, CONFIG_SIMULATOR, \
             CONFIG_STARTING_PORTFOLIO, CONFIG_SIMULATOR_FEES, CONFIG_EXCHANGES, CONFIG_TRADER
+        from octobot_evaluators.constants import CONFIG_FORCED_TIME_FRAME, CONFIG_FORCED_EVALUATOR
```

Comment:
As octobot_evaluator is not a "related module" as Trading is for Backtesting, an import of octobot_evaluators have to be cought in my opinion.

Reply:
They are, the calling method is doing it

Reply:
ok 👍